experiment/autobumper/main.go

```diff
@@ -0,0 +1,323 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/experiment/image-bumper/bumper"
+	"k8s.io/test-infra/prow/config/secret"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/robots/pr-creator/updater"
+)
+
+const prowPrefix = "gcr.io/k8s-prow/"
+const testImagePrefix = "gcr.io/k8s-testimages/"
+const prowRepo = "https://github.com/kubernetes/test-infra"
+const testImageRepo = prowRepo
+const oncallAddress = "https://storage.googleapis.com/kubernetes-jenkins/oncall.json"
+const githubOrg = "kubernetes"
+const githubRepo = "test-infra"
+
+func cdToRootDir() error {
+	if bazelWorkspace := os.Getenv("BUILD_WORKSPACE_DIRECTORY"); bazelWorkspace != "" {
+		if err := os.Chdir(bazelWorkspace); err != nil {
+			return fmt.Errorf("failed to chdir to bazel workspace (%s): %v", bazelWorkspace, err)
+		}
+	}
+	cmd := exec.Command("git", "rev-parse", "--show-toplevel")
```

Comment:
This could be in an else, right? This could also be handled by the hack entry script as well, potentially

Reply:
Left it in, but made it return early above.